digits/utils/image.py

```diff
@@ -35,6 +49,27 @@
 # List of supported file extensions
 # Use like "if filename.endswith(SUPPORTED_EXTENSIONS)"
 SUPPORTED_EXTENSIONS = ('.png','.jpg','.jpeg','.bmp','.ppm')
+if dicom is not None:
+    SUPPORTED_EXTENSIONS = SUPPORTED_EXTENSIONS + dicom_extension
+if nifti is not None:
+    SUPPORTED_EXTENSIONS = SUPPORTED_EXTENSIONS + nifti_extension
+
+def load_image_ex(path):
+    """
+    Handles images not recognized by load_image
+    Reads a file from `path` and returns a PIL.Image with mode 'F' (float32)
+    Raises LoadImageError
+
+    Arguments:
+    path -- file system path to the image
+    """
+    try:
+        dm = dicom.read_file(path)
+    except dicom.errors.InvalidDicomError:
+        raise errors.LoadImageError, 'Invalid Dicom file'
+    pixels = dm.pixel_array
+    image = PIL.Image.fromarray(pixels.astype(np.float))
```

Comment:
I think we can support that in the future.  Adding 3D/4D data support from one single DICOM file would involve many modifications.  Maybe that's because my limited understanding on the current codes.